### DISCUSSION TITLE
Boundary trace plotting on pole figure

### DISCUSSION THREAD

**Discussion by ddimitriou2009** [mtex-toolbox/mtex#2178] [Ask Anything] [answered]
Hello everyone,
I am trying to plot my tilt low angle boundary trace as a plane on the pole figure plot. I have tried a few things but could not make it work. 
Does anyone have any ideas on how to fix it?
Thanks in advance

**Comment by kilir** [comment, marked as answer]
Hi,
the trace of of type vector3d, so somethign like this 
```Matlab
tr = vector3d(4,2,0)
plot(cross(v,zvector),'circle')
```
should do.

Cheers,
Rüdiger
